olm-catalog/serverless-operator/1.6.0/serverless-operator.v1.6.0.clusterserviceversion.yaml

```diff
@@ -91,35 +91,96 @@ spec:
       version: v1alpha1
   minKubeVersion: 1.15.0
   description: |-
-    The Red Hat Serverless Operator provides a collection of API's to
-    install various "serverless" services.
+    The Red Hat OpenShift Serverless Operator provides a collection of APIs that
+    enables containers, microservices and functions to run "serverless".
+    Applications can scale up and down (to zero) on demand and be triggered by a
+    number of event sources. OpenShift Serverless integrates with a number of
+    platform services, such as Metering and Monitoring and it is based on the open
+    source project Knative.  
 
     This is a **[Tech Preview release](https://access.redhat.com/support/offerings/techpreview)!**
 
-    # Knative Serving
+    ## Prerequisites
+
+    The Serverless operator's provided APIs such as Knative Serving have certain
```

Comment:
Stay consistent with naming please :slightly_smiling_face: i.e. always use OpenShift Serverless

I'd rephrase this to make it easier to digest, maybe:

The APIs provided with the OpenShift Serverless operator require minimum cluster sizes on OpenShift Container Platform. For more information, see the documentation on [Getting started with OpenShift Serverless](https://access.redhat.com/documentation/en-us/openshift_container_platform/4.3/html-single/serverless_applications/index#serverless-getting-started).